devops/chef/README.md

```diff
@@ -1,22 +1,42 @@
-Chef Cookbook Best Practices
-============================
+# Chef Cookbook Best Practices
+
+## General
 
   1. Create a cookbook for every service or application that you have.
-  2. Create a public recipe for each component of the service or application (app server, db server, etc.)
-  3. Create private recipes for repeatable stuff you want to put in every cookbook
+  1. Create a public recipe for each component of the service or application (app server, db server, etc.)
+  1. Create private recipes for repeatable stuff you want to put in every cookbook
     (not exposed to end user, not in run_list of a node, included in other recipes, documented)
-  4. Create data driven cookbooks - Use attributes so the behavior can be changed at runtime.  Databags and encrypted databags can also be used if needed for organization level configuration.  Create an attributes file for each recipe.  Configurable per environment.
-  5. Use default recipe so cookbook works out of the box.
-  6. Library cookbooks - not really for recipes
-  7. Wrapper cookbooks - lightweight - contain mostly attribute overrides.
-  8. Iterate often. Make small changes. Use Vagrant VMs to quickly test your changes. `vagrant plugin install vagrant-berkshelf`
-  9. Lock down cookbook versions per environment.
-  10. Use custom attributes in a wrapper cookbook instead of using roles.
+  1. Create data driven cookbooks - Use attributes so the behavior can be changed at runtime.  Databags and encrypted databags can also be used if needed for organization level configuration.  Create an attributes file for each recipe.  Configurable per environment.
+  1. Use default recipe so cookbook works out of the box.
+  1. Library cookbooks - not really for recipes
+  1. Wrapper cookbooks - lightweight - contain mostly attribute overrides.
+  1. Iterate often. Make small changes.
```

Comment:
Is this important? Does it fit in this list? Maybe it does, but it doesn't feel like a pearl..